video/external/src/decoder/README.md

```diff
@@ -0,0 +1,10 @@
+To update to a new OpenH264 release:
```

Comment:
TODO: This should probably have a GitHub Actions workflow to check for and apply the latest release.

Reply:
That's a great idea, I just couldn't quite figure out how to best handle the built-in hardcoded library version matching sanity check - the `== (2, 4, 1)` part.